README.md

```
@@ -2,313 +2,1523 @@
+- [Pranshu Patel](https://pranshu05.vercel.app)
+- [Prashant Khandelwal](https://prashantk.dev)
+- [Prataya Silla](https://prataya-portfolio.vercel.app)
+- [Praveen Kumar Purushothaman](https://praveen.science) [Full Stack React Js Developer | Director Of Engineering]
+- [Praveen Kumar](https://praveengongada.com) [Software Development Engineer]
+- [Preet Suthar](https://preetsuthar.me)
+- [Prem Acharya](https://premacharya.vercel.app)
+- [Prem Prakash Sharma](https://premprakashsharma.vercel.app)
+- [Pritam Debnath](https://pridebnath.github.io/portfolio-v2) [Frontend Developer]
+- [Pritu Yadav](https://prituyadav.github.io)
+- [Priya Nayak](https://priya180975.github.io/portfolio)
+- [Priyabrata Singh](https://pmanifestoo.vercel.app)[Full Stack Developer | Ux/Ui Designer]
```

markdownlint-cli2 (0.18.1)

Reversed link syntax
(https://pmanifestoo.vercel.app)